### PR TITLE
Add Java 21 jre ubi9 minimal image

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

File: ga/latest/kernel/Dockerfile.ubi.openjdk21 [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_BUILD_LABEL={replace_with_gm_driver_label}
+
+LABEL org.opencontainers.image.authors="Leo Christy Jesuraj, Thomas Watson, Wendy Raschke, Michal Broz" \
+      org.opencontainers.image.vendor="IBM" \
+      org.opencontainers.image.url="https://github.com/WASdev/ci.docker" \
+      org.opencontainers.image.documentation="https://www.ibm.com/support/knowledgecenter/SSAW57_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/cwlp_about.html" \
+      org.opencontainers.image.version="$LIBERTY_VERSION" \
+      org.opencontainers.image.revision="$LIBERTY_BUILD_LABEL" \
+      org.opencontainers.image.description="This image contains the WebSphere Liberty runtime with IBM Semeru Runtime Open Edition OpenJDK with OpenJ9 and Red Hat's UBI 9 as the base OS.  For more information on this image please see https://ibm.biz/wl-app-image-template" \
```
Comment (leochr) [inline ga/latest/kernel/Dockerfile.ubi.openjdk21:56]: UBI 9 --> UBI 9 minimal 

Same comment for the summary and description labels below

**Review by leochr — COMMENTED** [review]
@tjwatson Looks good. Just a minor comment. Thank you

File: ga/latest/kernel/Dockerfile.ubi.openjdk21 (hunk 1 above)
Comment (tjwatson) [reply ga/latest/kernel/Dockerfile.ubi.openjdk21]: done

**Comment by tjwatson** [comment]
> @tjwatson Thanks for the update. The Java 21 Dockerfiles for can be copied over to the `24.0.0.1` directory. kernel-slim can be copied over as is. But the base/parent image for _full_ needs to append the version `24.0.0.1-` to the beginning of the tag.

Done

**Review by leochr — DISMISSED** [review]

**Review by mbroz2 — APPROVED** [review]

**Review by mbroz2 — DISMISSED** [review]

**Review by leochr — APPROVED** [review]

**Pull request merged**